test/functional/test-cid.js

```diff
@@ -596,8 +613,7 @@ describe('cid', () => {
         cookieName: 'cookie_name',
       }, hasConsent).then(c => {
         expect(c).to.exist;
-        expect(c).to
-            .equal('amp-sha384([1,2,3,0,0,0,0,0,0,0,0,0,0,0,0,15])');
+        expect(c).to.equal('amp-AQIDAAAAAAAAAAAAAAAADw');
```

Comment:
@lannka is right: when we recognize the AMP-generated cookie value in Google ads code, we expect it to match /amp-[\w-]{64}/.  That's what @avimehta told us was the desired behavior :-).